docs/markdown/Python/python/python-backend.md

```diff
@@ -20,6 +20,7 @@ Enable the Python [backend](doc:enabling-backends) like this:
 backend_packages = [
   "pants.backend.python"
 ]
+tailor_pex_binary_targets = false
```

Comment:
This goes in the `[python]` section

```suggestion

[python]
# This will become the default in Pants 2.15.
tailor_pex_binary_targets = false
```